doc-content/enterprise-only/admin-and-config/managing-business-central-artifacts-con.adoc

```diff
@@ -1,11 +1,11 @@
 [id='managing-business-central-artifacts-con']
-= Managing artifacts
+= Artifact management
 
 You can manage artifacts from the *Artifacts* page in {CENTRAL}. The artifact repository is a local Maven repository and there is only one Maven repository for each installation. {CENTRAL} recommends using Maven repository solutions like _Sonatype Nexus(TM)_, _Apache Archiva(TM)_, or _JFrog Artifactory(TM)_.
 
-The *Artifacts* page lists all the artifacts in the Maven repository. You can also upload a new artifact to the Maven repository using the *Upload* button.
+The *Artifacts* page lists all the artifacts in the Maven repository. You can also upload an artifact to the Maven repository using the *Upload* button.
 
 [NOTE]
 =====
-Only _*.jar_, _*.kjar_, or _pom.xml_ files can be uploaded to the Artifacts repository.
+Only JAR, KJAR, and `pom.xml` files can be uploaded to the *Artifacts* repository.
```

Comment:
Make second person?

Reply:
"You can only upload JAR, KJAR, and `pom.xml` files to the *Artifacts* repository."